src/main/java/frc/robot/commands/ClimberDisableCMD.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Climber2020;
+
+public class ClimberDisableCMD extends CommandBase {
+    private Climber2020 climber;
+
+    public ClimberDisableCMD(Climber2020 climber) {
+        this.climber = climber;
+
+        addRequirements(climber);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+    }
+
+    // Called every time the scheduler runs while the command is scheduled.
+    @Override
+    public void execute() {
+        climber.enable();
+    }
```

Comment:
Since it is only run once, would it be more expressive to put this in initialize instead?